defaults/genesis.json

```diff
@@ -149,6 +166,75 @@
           "destination": "ed0120CE7FA46C9DCE7EA4B125E2E36BDB63EA33073E7590AC92816AE1E861B7048B03@wonderland"
         }
       }
+    },
+    {
+      "Register": {
+        "Trigger": {
+          "id": "multisig_domains",
+          "action": {
+            "executable": {
+              "Wasm": "cb253c4ebe8ae68ad6591b4dd7acc447282df663fc16a3cc20f9938bfa0d3d83"
+            },
+            "repeats": "Indefinitely",
+            "authority": "ed01201F677E0900C2F633391310D12D155112DF65EDF9DC800D13797CEE5DAF47B890@system",
+            "filter": {
+              "Data": {
+                "Domain": {
+                  "id_matcher": null,
+                  "event_set": [
+                    "Created",
+                    "OwnerChanged"
+                  ]
+                }
+              }
+            },
+            "metadata": {}
+          }
+        }
+      }
+    },
+    {
+      "Grant": {
+        "Permission": {
+          "object": {
+            "name": "CanRegisterAnyTrigger",
+            "payload": null
+          },
+          "destination": "ed01201F677E0900C2F633391310D12D155112DF65EDF9DC800D13797CEE5DAF47B890@system"
+        }
+      }
+    },
+    {
+      "Grant": {
+        "Permission": {
+          "object": {
+            "name": "CanUnregisterAnyTrigger",
+            "payload": null
+          },
+          "destination": "ed01201F677E0900C2F633391310D12D155112DF65EDF9DC800D13797CEE5DAF47B890@system"
+        }
+      }
+    },
+    {
+      "Register": {
+        "Trigger": {
+          "id": "multisig_accounts_wonderland",
+          "action": {
+            "executable": {
+              "Wasm": "6f94f34b715a40aa0b98ea3a6e83107844e55de1625de6b6dbbfdec125a1ad07"
+            },
+            "repeats": "Indefinitely",
+            "authority": "ed0120CE7FA46C9DCE7EA4B125E2E36BDB63EA33073E7590AC92816AE1E861B7048B03@wonderland",
+            "filter": {
+              "ExecuteTrigger": {
+                "trigger_id": "multisig_accounts_wonderland",
+                "authority": null
+              }
+            },
+            "metadata": {}
+          }
+        }
+      }
     }
```

Comment:
I'd change this approach to fit #5123 since the wasm samples is superset of the genesis trigger blobs.

Instead of interpreting trigger registration as an ordinary instruction, we can give the genesis triggers a special field in `genesis.json` just like the executor, as like @mversic originally suggested:

```json
{
  "executor": "./executor.wasm",
  "wasm_triggers": [
    "multisig_accounts_wonderland": {
      "executable": "multisig_accounts",
      "repeats": "Indefinitely",
      "authority": "ed0120CE7FA46C9DCE7EA4B125E2E36BDB63EA33073E7590AC92816AE1E861B7048B03@wonderland",
      "filter": {
        "ExecuteTrigger": {
          "trigger_id": "multisig_accounts_wonderland",
          "authority": null
        }
      },
      "metadata": {}
    }
  ]
}
```

then `executable: &str` will be `WasmSmartContract` via `load_sample_wasm()` to compose a `Register<Trigger>` instruction.

Note that you will need to precompile wasm samples not only on integration tests, but also whenever genesis is needed